### DISCUSSION TITLE
complete event title on mouseover/hover in standard title-tag

### DISCUSSION THREAD

**Discussion by inu-web** [vkurko/calendar#342] [Q&A] [answered]
Is there a standard way of showing the title on mouseover/hovering an event
in my timeline/grid views the events are too small to show an proper amount of characters of the events titles - so having the (complete) title string in the html tag "title" for mouseover would be great.
thanks in advance for any hint!

Gerald

**Comment by vkurko** [comment, marked as answer]
There is no option to set the title attribute _out of the box_. But you have full control over the content displayed inside the event using the [eventContent](https://github.com/vkurko/calendar#eventcontent) option. Perhaps this option will suit your needs.

**Comment by inu-web** [comment]
Thank you for the hint, I was already looking into the "eventContent" and "resourceLabelContent" options before because i thought about just adding a title tag there on some wrapper element.
Could you help me to find the definition of the default content that is used if this options are not set or if they are "undefined".
Then I could "mimic" the "default content" of this two options inside a custom div wrapper with a title tag easily.

**Reply by vkurko** [reply]
[Here](https://github.com/vkurko/calendar/blob/da6900c50b70a00af1bb073952c1ec06e7d2bdb9/packages/core/src/lib/events.js#L100) is the default content. I hope everything will be clear from the code.

**Reply by inu-web** [reply]
Thank you very much for the link - everything clear now - works perfect 🙏

**Reply by coolblades** [reply]
another solution is use:
eventMouseEnter: e => {
                    $(e.el).attr('title', e.event.title);
}

**Reply by jinhuang11** [reply]
Thanks @vkurko @inu-web @coolblades for sharing the solutions.
I was not able to use jQuery, and the following vanilla JavaScript works for my svelte project: 

    eventMouseEnter: function (info) {
      info.el.setAttribute("title", info.event.title);
    },
